### DISCUSSION TITLE
MGI single cell rna-seq as input of seurat

### DISCUSSION THREAD

**Discussion by Zoeyau531** [satijalab/seurat#9011] [General]
Dear All,

I would like to know if I can input MGI single cell rna-seq data into Seurat for analysis?

I only used 10X data for seurat before.

What will be the "filtered_feature_bc_matrix" which includes barcodes.tsv.gz, features.tsv.gz and matrix.mtx.gz in MGI dnbc4tools output?

thanks!

**Comment by wudustan** [comment]
You can create a Seurat object from any kind of matrix that is structured as *features* x *cells*. 

`mgi.seurat <- CreateSeuratObject(mgi.counts, assay = "RNA")`

It looks like MGI have given some basic guidance on how to work with common single cell analysis software [here](https://github.com/MGI-tech-bioinformatics/DNBelab_C_Series_HT_scRNA-analysis-software/blob/version2.0/doc/io.md)
